start_bs.py

```diff
@@ -125,6 +125,7 @@ class SampleXYZ(Device):
 
     from embl_robot import EMBLRobot
     robot = EMBLRobot()
+    robot.canExchangeSamples = False
```

Comment:
you would need to make the EMBL robot an Ophyd robot or something to be able to get the `canExchangeSamples` variable

Reply:
change added to embl code to handle the function required 671207e